wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1183,6 +1191,13 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
     }
 
+    private func tearDownActiveSession(completion: (() -> Void)?) {
+        activeUserSession = nil
+        completion?()
+
+        // TODO: what about background sessions and observers?
```

Comment:
1) by background Sessions do you mean other accounts sessions? If yes, nothing to do.
2) which observers are you talking about?

Reply:
There are methods in this file which seem to be related to creating and tearing down a user session, like:
- `startBackgroundSession(...)` and `tearDownBackgroundSession(...)`
- `registerObservers()` and `tearDownObservers(...)`